compat/src/suspense.js

```diff
@@ -119,14 +127,9 @@ Suspense.prototype._childDidSuspend = function(promise, suspendingVNode) {
 		}
 	};
 
-	suspendingComponent._suspendedComponentWillUnmount =
-		suspendingComponent.componentWillUnmount;
-	suspendingComponent.componentWillUnmount = () => {
+	suspendingComponent._onResolve = () => {
+		suspendingComponent._onResolve = null;
 		onResolved();
-
-		if (suspendingComponent._suspendedComponentWillUnmount) {
-			suspendingComponent._suspendedComponentWillUnmount();
-		}
 	};
```

Comment:
I think this could be simplified to:

```js
	suspendingComponent._onResolve = onResolved;
```

... as long as the following was added around L112:

```js
		suspendingComponent._onResolve = null;
```

Reply:
got it! updated!